Plugins/AlarmClock/AlarmClockPlugin.cs

```diff
@@ -0,0 +1,60 @@
+﻿using BasePlugin.Interfaces;
+using BasePlugin.Records;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AlarmClock
+{
+    public class AlarmClockPlugin : IPluginWithScheduler
+    {
+        IScheduler _scheduler;
+        public AlarmClockPlugin(IScheduler scheduler) => _scheduler = scheduler;
+
+        public static string _Id = "Alarm-Clock";
+        public string Id => _Id;
+        public PluginOutput Execute(PluginInput input)
+        {
+            DateTime dateTime = DateTime.Now;
+            var sh = input.Message.Split(':').First();
+            var sm = input.Message.Split(':').Skip(1).First();
+
+            if (double.TryParse(sh, out double a) && double.TryParse(sm, out double b))
```

Comment:
1. You want double or int?
2. Do you use the output afterword's? - if so, save it